src/ui/components/results/directanswercomponent.js

```diff
@@ -60,7 +67,10 @@ export default class DirectAnswerComponent extends Component {
      * The display text for the View Details click to action link
      * @type {string}
      */
-    this._viewDetailsText = config.viewDetailsText || 'View Details';
+    this._viewDetailsText = config.viewDetailsText || TranslationFlagger.flag({
+      phrase: 'View Details',
+      context: 'Button label, call to action'
+    });
```

Comment:
What do you think about the context: 'Button label, view is a verb'? The translator might not be familiar with a call to action.